src/ratings/RatingStore.ts

```diff
@@ -0,0 +1,256 @@
+import { Rating } from '../types/Rating';
+import { reportError } from '../drivers/SentryConnector';
+import { RatingDataStore } from './interfaces/RatingDataStore';
+import { Db, ObjectId } from 'mongodb';
+import { MongoDriver } from '../drivers/MongoDriver';
+import { ServiceError, ServiceErrorReason } from '../errors';
+import { UserInfo } from '../types/UserInfo';
+
+
+enum Collections {
+  RATINGS = 'ratings',
+}
+
+export class RatingStore implements RatingDataStore {
+  private static instance: RatingDataStore;
+  private db: Db;
+
+  private constructor() {
+      this.db = MongoDriver.getConnection();
+  }
+
+  static getInstance(): RatingDataStore {
+      if (!RatingStore.instance) {
```

Comment:
Should we use `this` here instead of `RatingStore`? I see in a later section of these changes that `this` is used in a static method over `RatingStore`.
```suggestion
      if (!this.instance) {
```